apm-agent-plugins/apm-profiling-plugin/src/main/java/co/elastic/apm/agent/profiler/CallTree.java

```diff
@@ -735,6 +754,8 @@ public void resetState() {
             super.resetState();
             activeSpan = null;
             activationTimestamp = -1;
+            // todo - should we reset activeSpanSerialized?
```

Comment:
leftover TODO ?

Reply:
Well, half leftover...
Didn't want to look into it now, but wanted the reminder. But I will see if we need to something with it or leave it as is